sema/query/__main__.py

```
@@ -17,26 +17,14 @@
-        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
-    )
-
-    parser.add_argument(
-        "-l",
-        "--logconf",
-        type=str,
-        action="store",
-        help="location of the logging config (yml) to use",
+    parser = SemaArgsParser(
+        "Py Project to extra table data from "
+        "knowledge-graphs using sparql templates"
```

Ruff

Trailing comma missing

Add trailing comma

(COM812)